boards/xtensa/esp32s2_lolin_mini/esp32s2_lolin_mini_defconfig

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_XTENSA_RESET_VECTOR=n
+
+CONFIG_BOARD_ESP32S2_LOLIN_MINI=y
+CONFIG_SOC_ESP32S2=y
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=240000000
```

Comment:
You can keep this, but it is no required.

Reply:
Can I remove all of them or just line 9? Thanks!

Reply:
Just line 9.

Reply:
Done